### ISSUE TITLE
Support displaying GIFs with `preview()` in Juno's Plot pane

### ISSUE THREAD

**Issue by mroavi** [JuliaGraphics/Luxor.jl#84]
The `preview()` documentation mentions 

> If working in Juno, display a PNG or SVG file in the Plot pane.

It would be great if `preview()` could also be used to display GIFs. I know that the animation functionality of the Plots.jl package is capable of displaying GIFs in the Juno's Plot pane.

**Comment by cormullion** [comment]
Good idea! I've added an automatic preview in the Juno Plots pane of a GIF made by `animate()` in the latest commit (https://github.com/JuliaGraphics/Luxor.jl/commit/4e2848197f4e7e09d08ddca63548e057eb09c91d). For some reason it doesn't work in a Jupyter notebook...

**Comment by mroavi** [comment]
Wow, that was quick. It works great on my setup! 

Just to be clear for anybody else reading this: an automatic preview of GIFs in the Juno Plots was implemented (i.e. the GIF automatically displays in Juno's Plot pane if `animate(...)` was the last function invoked). The behavior of the `preview()` function stayed the same.

I will close the issue since the functionality that I proposed does work fine.
